proposals/utils/Constants.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.18;
+
+library Constants {
+    /// @notice Arbitrum mainnet chainid
+    uint256 public constant ARBITRUM_MAINNET = 42161;
+
+    /// @notice Arbitrum sepolia testnet chainid
+    uint256 public constant ARBITRUM_TESTNET = 421614;
```

Comment:
Maybe I'm just being picky, but perhaps we should explicitly call these `ARBITRUM_ONE` and `ARBITRUM_SEPOLIA` ?

Reply:
Fair enough. Lets mark this as resolved.

Reply:
I agree with Elliot, although I am ok with one/sepolia as well. @intl-man please let me know if you want to update to one/sepolia.